src/OrbitGl/TrackManager.cpp

```diff
@@ -464,6 +463,15 @@ ThreadTrack* TrackManager::GetOrCreateThreadTrack(uint32_t tid) {
   return track.get();
 }
 
+std::optional<ThreadTrack*> TrackManager::GetThreadTrack(uint32_t tid) {
```

Comment:
This function should be marked `const`

Reply:
Should it be given const when I'm accessing the map with the `[]` operator?

Reply:
Yes. It's rather the other way around. A function should be made const if it's possible to implement it in a non-mutating way. And that means you can't use operator[].

So it's the other way around. Making the function const is the priority and it dictates that you can't use operator[].